frontend/src/components/Apps/App/App.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './App.module.css';
+
+interface AppInformation {
+	information: string[];
+}
+
+const App: React.FunctionComponent<AppInformation> = ({
+	information,
+}: AppInformation) => {
+	const [title, description, owners] = information;
```

Comment:
This leaves a little bit to be desired as the information array may contain title, description, and owners but could be extended and thus have to be changed in multiple places; could we strongly type this `information` (possibly rename it to something better as well) into an object containing this instead?

Reply:
Completely agree. Would destructuring the prop and typing those with appropriate names work instead?

Reply:
I think the AppInformation could just contain title, description, owners and we can just access it there.